.tools/New-WikiFolder.ps1

```diff
@@ -1,85 +1,121 @@
-        } elseif ($line -match '\.OUTPUTS') {
-            $markdownOutput += '## Outputs'
-        } elseif ($line -match '\.LINK') {
-            $markdownOutput += '## Link'
-        } elseif ($line -match '\.PARAMETER') {
-            $markdownOutput += '### ' + ($line -replace '\.PARAMETER', '')
-        } elseif ($line -match '\.EXAMPLE') {
-            $markdownOutput += "### Example $exampleCounter"
-            $exampleCounter++
-        } else {
-            $markdownOutput += $line -replace '<#', '' -replace '#>', ''
-        }
-    }
-
-    return ($markdownOutput -join "`n").Trim()
+	return $markdownContent
 }
-
-[string]$scriptDirectory = Split-Path -Path $MyInvocation.MyCommand.Definition -Parent
-
-if ($false -eq (Test-Path "$scriptDirectory\AppDeployToolkitExtensions.ps1")) {
-    Write-Warning "AppDeployToolkitExtensions.ps1 not found, please place this script next to AppDeployToolkitExtensions.ps1"
-    Read-Host -Prompt "Press Enter to continue, CTRL+C to abort"
+try {
+	[System.IO.FileInfo]$extensionFile = Get-ChildItem -Filter 'AppDeployToolkit\AppDeployToolkitExtensions.ps1' -Recurse -Depth 1 -Path "$PSScriptRoot\..\" -ErrorAction SilentlyContinue
+	[System.Management.Automation.Language.ScriptBlockAst]$ast = [System.Management.Automation.Language.Parser]::ParseFile($extensionFile.FullName, [ref]$null, [ref]$null)
+}
+catch {
+	Write-Warning 'AppDeployToolkitExtensions.ps1 not found, please place this script in the package directory'
+	Read-Host -Prompt 'Press Enter to continue, CTRL+C to abort'
 }
 
-[string]$scriptContent = Get-Content -Path "$scriptDirectory\AppDeployToolkitExtensions.ps1" -Raw
-[psobject]$ast = [System.Management.Automation.Language.Parser]::ParseInput($scriptContent, [ref]$null, [ref]$null)
+New-Item -Name 'wiki' -ItemType Directory -Force -ErrorAction SilentlyContinue
 
-New-Item -Name "wiki" -ItemType Directory -Force -ErrorAction SilentlyContinue
+[System.Management.Automation.Language.FunctionDefinitionAst[]]$functionDefinitions = $ast.FindAll(
+	{
+		$args[0] -is [System.Management.Automation.Language.FunctionDefinitionAst]
+	},
+	$false
+)
 
-[psobject]$functionDefinitions = $ast.FindAll({ $args[0] -is [System.Management.Automation.Language.FunctionDefinitionAst] }, $true)
 foreach ($functionDefinition in $functionDefinitions) {
-    [string]$functionName = $functionDefinition.Name
-    [string]$summaryBlockPattern = '(?s)<#(.*?)#>'
-    [string]$summaryBlock = ([regex]::Matches($functionDefinition.Extent.Text, $summaryBlockPattern)).Value.Trim()
-
-    if (-not [string]::IsNullOrEmpty($summaryBlock)) {
-        [string]$markdownContent = ConvertTo-Markdown -Content $summaryBlock
-        Set-Content -Path "wiki\$functionName.md" -Value $markdownContent -Force
-    }
+	if ($null -ne $functionDefinition.GetHelpContent()) {
+		Set-Content -Path "wiki\$($functionDefinition.Name).md" -Value (ConvertTo-Markdown -HelpContent $functionDefinition.GetHelpContent()) -Force
```

Comment:
the Function Name changed to ConvertTo-CommentHelpToMarkdown

Reply:
Fixed